vclib/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/HolderAgent.kt

```diff
@@ -371,18 +371,21 @@ class HolderAgent(
         // MUST have the value $ (top level root path) when only one Verifiable Presentation is contained in the VP Token,
         // and MUST have the value $[n] (indexed path from root) when there are multiple Verifiable Presentations,
         // where n is the index to select.
-        path = index?.let { "\$[$it]" } ?: "\$"
+        path = index?.let { "\$[$it]" } ?: "\$",
     )
 }
 
 open class PresentationException(message: String) : Exception(message)
```

Comment:
Maybe move out exceptions to a different file, probably make it a `sealed class`?

Reply:
Decided to move `PresentationException` to class Holder, so that there is a common exception type for holder implementations (as it is an interface after all).

Also removed all the other exception types and removed them with instantiations of `PresentationException` itself as all of them were only used on one place anyway.. does that sound reasonable?

Done in commit c0220b8558f199dce5ca6c263e4aff84ce1a22c7